### PR TITLE
Replace "pragma" with "preprocessor directive".

### LINKED ISSUE #27911: NRT migration docs mention incorrect C# preprocessor directive
Document mentions `#pragma nullable disable` in the section "Plan your migration" ( https://docs.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies#plan-your-migration ), we tried this, doesn't work. Then realised this is probably an incorrect compiler directive and it should be `#nullable disable` ? (ref: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#nullable-context )

Cheers

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fc516159-d7e1-46ba-7d31-538a0b811491
* Version Independent ID: 99552131-8839-7c22-fcdc-2baa7722e756
* Content: [Update your codebase to use nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies)
* Content Source: [docs/csharp/nullable-migration-strategies.md](https://github.com/dotnet/docs/blob/main/docs/csharp/nullable-migration-strategies.md)
* Product: **dotnet-csharp**
* Technology: **csharp-null-safety**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #27911

### CODE REVIEW

**Review by tdykstra — APPROVED** [review]
preprocess --> preprocessor

File: docs/csharp/nullable-migration-strategies.md [hunk 1]
```diff
@@ -12,16 +12,16 @@ If your codebase is relatively small, you can turn on the [feature in your proje
 
 ## Plan your migration
 
-Regardless of how you update your codebase, the goal is that nullable warnings and nullable annotations are enabled in your project. Once you reach that goal, you'll have the `<nullable>Enable</nullable>` setting in your project. You won't need any of the pragmas to adjust settings elsewhere.
+Regardless of how you update your codebase, the goal is that nullable warnings and nullable annotations are enabled in your project. Once you reach that goal, you'll have the `<nullable>Enable</nullable>` setting in your project. You won't need any of the preprocess directives to adjust settings elsewhere.
```
Comment (BillWagner) [inline docs/csharp/nullable-migration-strategies.md:15]:
```suggestion
Regardless of how you update your codebase, the goal is that nullable warnings and nullable annotations are enabled in your project. Once you reach that goal, you'll have the `<nullable>Enable</nullable>` setting in your project. You won't need any of the preprocessor directives to adjust settings elsewhere.
```

File: docs/csharp/nullable-migration-strategies.md [hunk 2]
```diff
@@ -12,16 +12,16 @@ If your codebase is relatively small, you can turn on the [feature in your proje
...
-1. ***Nullable enable as the default***: Set this default when you're actively developing new features. You want all new code to benefit nullable reference types and nullable static analysis. Using this default means you must add a `#pragma nullable disable` to the top of each file. You'll remove that pragma as you begin addressing the warnings in that file.
-1. ***Nullable warnings as the default***: Choose this default for a two-phase migration. In the first phase, address warnings. In the second phase, turn on annotations for declaring a variable's expected *null-state*. Using this default means you must add a `#pragma nullable disable` to the top of each file.
+1. ***Nullable disable as the default***: *disable* is the default if you don't add a `Nullable` element to your project file. Use this default when you're not actively adding new files to the codebase. The main activity is to update the library to use nullable reference types. Using this default means you add a nullable preprocess directives to each file as you update its code.
```
Comment (BillWagner) [inline docs/csharp/nullable-migration-strategies.md:19]:
```suggestion
1. ***Nullable disable as the default***: *disable* is the default if you don't add a `Nullable` element to your project file. Use this default when you're not actively adding new files to the codebase. The main activity is to update the library to use nullable reference types. Using this default means you add a nullable preprocessor directives to each file as you update its code.
```

File: docs/csharp/nullable-migration-strategies.md [hunk 3]
```diff
@@ -12,16 +12,16 @@ If your codebase is relatively small, you can turn on the [feature in your proje
...
+1. ***Nullable enable as the default***: Set this default when you're actively developing new features. You want all new code to benefit nullable reference types and nullable static analysis. Using this default means you must add a `#nullable disable` to the top of each file. You'll remove that preprocess directives as you begin addressing the warnings in that file.
```
Comment (BillWagner) [inline docs/csharp/nullable-migration-strategies.md:20]:
```suggestion
1. ***Nullable enable as the default***: Set this default when you're actively developing new features. You want all new code to benefit nullable reference types and nullable static analysis. Using this default means you must add a `#nullable disable` to the top of each file. You'll remove that preprocessor directives as you begin addressing the warnings in that file.
```

File: docs/csharp/nullable-migration-strategies.md [hunk 4]
```diff
@@ -12,16 +12,16 @@ If your codebase is relatively small, you can turn on the [feature in your proje
...
+Enabling nullable as the default creates more up-front work to add the preprocess directives to every file. The advantage is that every new code file added to the project will be nullable enabled. Any new work will be nullable aware; only existing code must be updated. Disabling nullable as the default works better if the library is stable, and the main focus of the development is to adopt nullable reference types. You turn on nullable reference types as you annotate APIs. When you've finished, you enable nullable reference types for the entire project. When you create a new file, you must add the preprocess directives and make it nullable aware. If any developers on your team forget, that new code is now in the backlog of work to make all code nullable aware.
```
Comment (BillWagner) [inline docs/csharp/nullable-migration-strategies.md:24]:
```suggestion
Enabling nullable as the default creates more up-front work to add the preprocessor directives to every file. The advantage is that every new code file added to the project will be nullable enabled. Any new work will be nullable aware; only existing code must be updated. Disabling nullable as the default works better if the library is stable, and the main focus of the development is to adopt nullable reference types. You turn on nullable reference types as you annotate APIs. When you've finished, you enable nullable reference types for the entire project. When you create a new file, you must add the preprocessor directives and make it nullable aware. If any developers on your team forget, that new code is now in the backlog of work to make all code nullable aware.
```

**Pull request merged**